zerver/models/custom_profile_fields.py

```diff
@@ -174,9 +178,12 @@ def as_dict(self) -> ProfileDataElementBase:
             "order": self.order,
             "required": self.required,
             "editable_by_user": self.editable_by_user,
+            "use_in_mentions": self.use_in_mentions,
         }
         if self.display_in_profile_summary:
             data_as_dict["display_in_profile_summary"] = True
+        if self.use_in_mentions:
```

Comment:
This does not make sense since you alread set `use_in_mentions` above.

Would be good to confirm though if we want to include this value in the object if the value is `False`.